src/renderer/viz/expressions/Ramp.js

```diff
@@ -97,6 +97,20 @@ import { DEFAULT_RAMP_OTHERS } from './constants';
 * @function
 * @api
 */
+
+/**
+ * Ramp Class
+ *
+ * A mapping between an input (numeric or categorical) and an output (number, colors and/or images)
+ * This class is instanced automatically by using the `ramp` function. It is documented for its methods.
```

Comment:
Same phrasing

Reply:
?

Reply:
" instanced automatically" sounds weird to me. Saying something like "you can use the `ramp` function to generate instances of the `Ramp` class" sounds better to me